docs/20.07-attributes.md

```diff
@@ -10,103 +10,29 @@ parent: Usage
 #### Component Attributes
 
 Attributes declared on the components will be all be accessible through the `state`.
-If the property is initialized in the `this.state`, the attribute will be reactive:
-
-```html
-<x-user status="thinking 🤔"><x-user>
-```
-
-`status` will therefore be reactive and the _thinking 🤔_ attribute value will overwrite the _Happy 😄_ default status.
-
-⚠️ A property that is not declared in the `state` won't be reactive.
-
-These properties can be accessed through `this.getAttribute()` from within the component.
-After all, these components are just native! 🏡
-
-
-#### Slots

```

Comment:
Where is the slot documentation?

Reply:
Same as above comment, there is a [dedicated file](https://github.com/mlbiche/lego/blob/docs/proofreading/docs/20.08-slots.md)